tests/contrib/operators/test_mysql_to_gcs_operator.py

```diff
@@ -262,7 +262,7 @@ def test_schema_file(self, gcs_hook_mock_class, mysql_hook_mock_class):
 
         gcs_hook_mock = gcs_hook_mock_class.return_value
 
-        def _assert_upload(bucket, obj, tmp_filename, mime_type):
+        def _assert_upload(bucket, obj, tmp_filename, mime_type):  # pylint disable=unused-argument
```

Comment:
```suggestion
        def _assert_upload(bucket, obj, *args, **kwargs):
```
As suggested above, this approach needs no `pylint:disable` comment and moreover adds information "that some additional stuff can be passed to the function but I don't care".